lib/support/video/finetune/ffmpegRemoveOrangeAndAddText.js

```diff
@@ -48,7 +49,9 @@ module.exports = {
     ];
 
     // keeping old legacy videoRaw
-    const showTimer = options.videoRaw ? false : options.videoParams.addTimer;
+    const showTimer = options.videoRaw
+      ? false
+      : options.videoParams.addTimer || videoDefaults.addTimer;
```

Comment:
This is incorrect. showTimer will be set to true for the following case:
```
options = {
videoParams: {addTimer: false}
}
```

Reply:
yep is a known bug but I haven't reported it :D

Reply:
fixed in https://github.com/sitespeedio/browsertime/pull/396